CHANGES

```diff
@@ -46,6 +46,10 @@ Upcoming release: BREAKING
 * aarch32 VM fault messages now deliver original (untranslated) faulting IP in a hypervisor context, matching
   aarch64 behaviour.
 * Correct the minimum size of a scheduling context. This changes the value of `seL4_MinSchedContextBits`.
+* Changed how `gen_config.h` files are generated. Previously, they were generated at CMake configure time. Now, they
+  are generated at build time. For example, to manually build the kernel `gen_config.h` file after running `cmake`,
+  run `ninja gen_config/kernel/gen_config.h`. Both `${prefix}_Gen` and `${prefix}_Config` depend on
+  `${prefix}/gen_config.h`
```

Comment:
Can you run `ninja kernel_Config`?

Reply:
Ah, no you can not. I have modified this PR accordingly.